compiler/z/codegen/OMRInstOpCode.hpp

```diff
@@ -659,33 +657,18 @@ class InstOpCode: public OMR::InstOpCode
    uint64_t setsCC() {return (setsZeroFlag() || setsSignFlag() || setsOverflowFlag() || setsCompareFlag() || setsCarryFlag() || (properties[_mnemonic] & S390OpProp_SetsCC));}
    uint64_t readsCC() {return properties[_mnemonic] & S390OpProp_ReadsCC;}
 
-   uint64_t readsMultipleRegs() {return properties[_mnemonic] & S390OpProp_ReadsMultipleRegs;}
-   uint64_t setsMultipleRegs() {return properties[_mnemonic] & S390OpProp_SetsMultipleRegs;}
    uint64_t setsZeroFlag() {return properties[_mnemonic] & S390OpProp_SetsZeroFlag;}
    uint64_t setsSignFlag() {return properties[_mnemonic] & S390OpProp_SetsSignFlag;}
-   uint64_t setsCarryFlag() {return properties[_mnemonic] & S390OpProp_SetsCarryFlag;}
    uint64_t setsOverflowFlag() {return properties[_mnemonic] & S390OpProp_SetsOverflowFlag;}
    uint64_t setsCompareFlag() {return properties[_mnemonic] & S390OpProp_SetsCompareFlag;}
+   uint64_t setsCarryFlag() { return setsZeroFlag(); }
```

Comment:
Carry and zero flags are synonymous. All instructions that have either have both, so the flags can be merged. I've left the query here because depending on context we may be talking about either carry or zero flag. Even though they map to the same condition code from a code reading perspective it helps to talk about the correct flag depending on context.